src/rust/engine/workunit_store/src/lib.rs

```diff
@@ -59,6 +60,7 @@ pub enum WorkunitState {
 
 #[derive(Clone, Debug, PartialEq, Eq, Hash)]
 pub struct WorkunitMetadata {
+  pub level: LogLevel,
```

Comment:
What is the thinking on using the python-compatibility level rather than using [log::Level](https://docs.rs/log/0.4.8/log/enum.Level.html)? It sortof feels like leaving the `PythonLogLevel` struct as a compatibility struct and then using `log::Level` would avoid coupling things.

Reply:
Makes sense, might as well use `log::Level`